MaterialLib/MPL/Components/cCarbonDioxide.h

```diff
@@ -0,0 +1,34 @@
+/**
+ * \author Norbert Grunwald
+ * \date   12.09.2017
+ * \brief
+ *
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+#ifndef MATERIALLIB_MPL_COMPONENTS_CCARBONDIOXIDE_H_
```

Comment:
We switched to `#pragma once` some time ago... in https://github.com/ufz/ogs/pull/1627 there is a script to do it automatically, though I don't promise it will work out of the box.